textile/chat-features.textile

```diff
@@ -116,7 +116,7 @@ h4(#rooms-lifecycle-operations). Room Lifecycle Operations
 ** @(CHA-RL3a)@ @[Testable]@ If the room is already in the @RELEASED@ status, this operation is no-op.
 ** @(CHA-RL3b)@ @[Testable]@ If the room is in the @DETACHED@ status, then the room is immediately transitioned to @RELEASED@ and the operation succeeds.
 ** @(CHA-RL3j)@ @[Testable]@ If the room is in the @INITIALIZED@ status, then the room is immediately transitioned to @RELEASED@ and the operation succeeds.
-** @(CHA-RL3c)@ @[Testable]@ If the room is in the @RELEASING@ status, then the operation will return the result of the pending @release@ operation, or throw any exception that it throws.
+** @(CHA-RL3c)@ This specification point has been removed.
```

Comment:
Perhaps a slight reword might work here - the intended behaviour is "we're releasing, so atomically this operation doesn't run until the previous one completes", so perhaps:

@(CHA-RL3c)@ @[Testable]@ If the a release operation in process, the call will wait until the previous operation is complete before proceeding. For example, if the room releases successfully, then the subsequent call will resolve per @CHA-RL3a@.

WDYT?

Reply:
- Actually, as I mentioned in the [PR summary](https://github.com/ably/specification/pull/230#issue-2671661962), the release operation never throws any exception and will always reach the `RELEASED` state.
- So, the statement `If the release operation in process` is never true according to [CHA-RL7] : |
- This implies that, per CHA-RL7, new operations will always have `_releaseInProgress` set to false.
- Additionally, I am confident chat-js code will [never enter this block](https://github.com/ably/ably-chat-js/blob/ee554e3d19e2ec718a3b3c90974753ee05ad6b52/src/core/room-lifecycle-manager.ts#L799), of all tests we have as a part of chat-js test-suite. So, the check and related implementation is redundant.

Reply:
Not just for this spec point, but we shouldn't have `in progress` checks as a part of any other Room operations.